app/src/main/java/com/example/parkingsystemkotlin/mvp/view/ReservationParkingView.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.parkingsystemkotlin.mvp.view
+
+import android.app.Activity
+import com.example.parkingsystemkotlin.databinding.ActivityReserverParkingLotBinding
+import com.example.parkingsystemkotlin.fragments.DateReservationDialogFragment
+import com.example.parkingsystemkotlin.mvp.contract.ReservationParkingContract
+import com.example.parkingsystemkotlin.mvp.view.base.ActivityView
+import com.example.parkingsystemkotlin.utils.Constant
+
+class ReservationParkingView(activity: Activity, binding: ActivityReserverParkingLotBinding) :
+    ActivityView(activity), ReservationParkingContract.ReservationActivityView {
+
+    override fun showDialogDataPiker(startDatePiker: Boolean) {
+        activity?.let {
```

Comment:
you are not using the activity here, why do you add this?

Reply:
Doing